config/kubernetes/sig-release/teams.yaml

```diff
@@ -10,6 +10,7 @@ teams:
     - idvoretskyi # PM
     - nikhita # ContribEx
     - spiffxp # 1.14 RT Lead / Testing
+    - krmayankk # 1.14 RT Bug Triage
```

Comment:
this list needs to be sorted :P

Reply:
and you'll need to add in the members list

(only org admins can be in the maintainers list)